woocommerce/woocommerce-android
# Order Details: Shipment tracking card fix

Fixes #3350. The shipment tracking card is displayed when:

- order has either no products or at least one physical product
- order doesn't have shipping labels

The shipment tracking availability is saved in the site-specific preferences, which is reset after a site is changed. 

**To test:**
1. Go to a site that has the tracking extension
2. Go to Orders -> Tap on an order without shipping labels that doesn't have just virtual products
3. Verify the shipping tracking card is initially not shown and after a moment it appears
4. Go back to the order list
5. Open another order without shipping labels that doesn't have just virtual products
6. Verify the shipping tracking card is shown immediately
7. Switch to a site that doesn't have the shipping tracking extension
8. Go to Orders -> Tap on an order
9. Verify the shipping tracking card is never shown
